bob-common/src/data.rs

```diff
@@ -92,26 +94,44 @@ pub type VDiskId = u32;
 
 #[derive(Clone)]
 pub struct BobData {
-    inner: Vec<u8>,
+    inner: Bytes,
     meta: BobMeta,
 }
 
 impl BobData {
-    pub fn new(inner: Vec<u8>, meta: BobMeta) -> Self {
+    const TIMESTAMP_LEN: usize = 8;
+    
+    pub fn new(inner: Bytes, meta: BobMeta) -> Self {
         BobData { inner, meta }
     }
 
     pub fn inner(&self) -> &[u8] {
         &self.inner
     }
 
-    pub fn into_inner(self) -> Vec<u8> {
+    pub fn into_inner(self) -> Bytes {
         self.inner
     }
 
     pub fn meta(&self) -> &BobMeta {
         &self.meta
     }
+
+    pub fn to_vec(&self) -> Vec<u8> {
+        let mut result = self.meta.timestamp.to_be_bytes().to_vec();
+        result.extend_from_slice(&self.inner);
```

Comment:
The output length of vector is known (`TIMESTAMP_LEN + self.inner.len()`), thus it is better to create vector with preallocated capacity and call `extend_from_slice` two times.